docs/source/variables.rst

```diff
@@ -0,0 +1,48 @@
+Variables
+=========
+
+.. image:: _static/images/variables_initial.png
+
+This panel shows the variables available along with their properties.
+It also allows the user to select a variable for further analysis in
+other panels.It has two parts.
+
+1. Displayer
+------------
+
+.. raw:: html
+
+   <img src="_static/images/displayer.png" alt="Displayer" style="width:270px;height:180px;">
+
+
+This multiselect widget acts both as a displayer and selector. It
+displays all the variables present in the Dataset or DataArray.
+The variables which are coordinates are annotated with the  ‘📈’
+symbol. Upon selection of a variable, its properties are updated
+in the adjoining description section.
+
+2. Describer
+------------
+
+.. image:: _static/images/describer.png
+
+The describer or description section consists of two striped tables.
```

Comment:
remove "describer"

@rsignell-usgs 's point about the style of these tables still stands, they should have smaller text to match the variable selector. I wish each table had a title ("variable attributes", "global attributes")